### DISCUSSION TITLE
Center Vertical

### DISCUSSION THREAD

**Discussion by appinteractive** [xvrh/lottie-flutter#182] [General]
I have massive issues when trying to center a Lottie asset inside other elements (like FittedBox). I would like to use it as a Splash animation which is centered vertically and fit the width. But I'm unable to center it vertically. Nothing works so far. The align property of Lottie.asset is ignored completely.

Any suggestions that does not involve calculating height differences?

**Comment by xvrh** [comment]
Hi, can you post some code?
Also, does your code work with a normal `Image.asset`?
